ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/logging-format.yml

```diff
@@ -0,0 +1,7 @@
+### Logging ###
+logging:
```

Comment:
do we need another "host" for this test? Instead of just using `logging` ?

Reply:
@ClausHolbechArista It's up to you. I can also add the formating to "logging". We just have to consider that the formating is global. So our test will either cover special formating or no cover it, if we go with only one host in the molecule test.

What do you think?

Reply:
Please add it to the `logging` host instead of a new host. We have many other test cases where logging is used, but this knob will not be set. Thanks

Reply:
will do, coming up soon!